src/renderer/features/network/ManageCustomRpcNode/ui/AddCustomRpcModal.tsx

```diff
@@ -0,0 +1,106 @@
+import { FormEvent, useEffect } from 'react';
+import { useForm } from 'effector-forms';
+import { useUnit } from 'effector-react';
+
+import { BaseModal, Button, Input, InputHint, Alert } from '@shared/ui';
+import { useI18n } from '@app/providers';
+import { OperationTitle } from '@entities/chain';
+import { RpcCheckResult, addCustomRpcModel } from '@features/network';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+export const AddCustomRpcModal = ({ isOpen, onClose }: Props) => {
+  const { t } = useI18n();
+  const rpcCheckResult = useUnit(addCustomRpcModel.$rpcConnectivityResult);
+  const isNodeExist = useUnit(addCustomRpcModel.$isNodeExist);
+  const network = useUnit(addCustomRpcModel.$selectedNetwork);
+  const isLoading = useUnit(addCustomRpcModel.$isLoading);
+
+  useEffect(() => {
+    addCustomRpcModel.events.formInitiated();
+  }, []);
+
+  const {
+    fields: { name, url },
+    submit,
+    isValid,
+    isDirty,
```

Comment:
Do we really need `isDirty`?

Reply:
for me yes, so that the button is disabled unless users changed any field.